lib/new_relic/agent/error_filter.rb

```diff
@@ -29,7 +29,8 @@ def load_all
       def load_from_config(setting, value = nil)
         errors = nil
         new_value = value || fetch_agent_config(setting.to_sym)
-        return if new_value.nil? || new_value.empty?
+
+        return if new_value.nil? || (new_value.respond_to?(:empty?) && new_value.empty?)
```

Comment:
Great pivot from yesterday! `respond_to?` is such a handy method. This is a common idiom in Ruby.